modules/enterprise/server/jar/src/main/java/org/rhq/enterprise/server/content/ContentManagerRemote.java

```diff
@@ -291,4 +291,7 @@ PackageTypeAndVersionFormatComposite findPackageTypeWithVersionFormat(Subject su
      */
     PackageVersion createPackageVersionWithDisplayVersion(Subject subject, String packageName, int packageTypeId,
         String version, String displayVersion, Integer architectureId, String temporaryContentHandle);
+
+
+    void removeHistoryDeploymentsBits();
```

Comment:
As stated in the removeHistoryDeploymentsBits() impl, shouldn't this require authentication (Subject)?